src/Deprecated/Conversion.UnitTests/XMakeConversionUnitTests.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="dogfood" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
is dogfood relevant here? it's deprecated so...meh

Reply:
Do deprecated unit tests even run? Like ever? Should we just delete this?

Reply:
they don't at the moment but I hate to delete them because we _should_ turn them back on (but maybe we can just delete the code first?

Reply:
More than happy to delete deprecated UTs if that's the decision. But for another PR, as this one is big enough :)